runtime/dart_init.cc

```diff
@@ -92,10 +92,8 @@ namespace {
 
 // Arguments passed to the Dart VM in all configurations.
 static const char* kDartLanguageArgs[] = {
-    "--enable_mirrors=false",
-    "--background_compilation",
-    "--await_is_keyword",
-    "--causal_async_stacks",
+    "--enable_mirrors=false", "--background_compilation", "--await_is_keyword",
+    "--causal_async_stacks",  "--limit-ints-to-64-bits",
```

Comment:
I'm assuming this was the result of `clang-format`?

Reply:
Yes.